### PR TITLE
create quest scene

### PR DESCRIPTION
**Pull request by alexcsoong**
PR TITLE
Create quest scene

## Description
I created the quest scene which is in Branches -> Alex -> Assets -> Scenes. Image of it can be found in #updates in Discord (sent 3/4/24).

## Motivation and Context
We need a scene to have the quest in.

## How Has This Been Tested?
I viewed it in the camera.

## Screenshots (if appropriate):
image

### CODE REVIEW

**Review by sakshikakkad — CHANGES_REQUESTED** [review]
the quest scene looks good, when you're setting up the scene make sure that the trees aren't placed on as individual prefabs, it's a lot cleaner to use paint details and do it with a brush. if you are going to use prefabs it needs to be a bit more cleanly organized in the hierarchy (I would nest all trees under an empty game object, and I wouldn't have some nested under the terrain).
image

The asset level hierarchy has the spiders package you downloaded, which is the main reason I can't approve the PR. Make sure with whatever you download, things get sorted into the correct folders. Until we officially merge your changes to main, I would just keep the spiders folder in your own branch and I will sort it to the right places when I merge.

image

I would take these trees out because they are not rendering correctly. You might need to tinker with the materials/textures or if you don't want to deal with it then just take them out.

image

Last thing is the scale of this scene is much bigger than main or garden. All the assets (trees, painted textures, etc) that I see should all be fitting on the center square for consistency.

image

**Comment by alexcsoong** [comment]
``

**Pull request closed without merging**
